InterviewFlashCards/QueryManager.swift

```diff
@@ -9,17 +9,21 @@
 import Foundation
 import Firebase
 
-enum RequestType: Int {
+public protocol Networking {
+    func getData(for requestType: RequestType, completion: ([[NSObject: AnyObject]]) -> Void)
+}
+
+public enum RequestType: Int {
     case iOS
     case DataStructures
     case Algorithms
 }
 
-class QueryManager: NSObject {
-    
+class QueryManager: Networking {
```

Comment:
yeah but you can use `:` to mean both conformance to a protocol or subclassing. If the thing after the `:` is a class, then it would mean subclassing.